src/Upgrade/Application/Dto/StepsResponseDto.php

```diff
@@ -85,6 +85,26 @@ public function getOutputMessage(): ?string
         return implode(PHP_EOL, $this->outputMessageList);
     }
 
+    /**
+     * @return array<string>
+     */
+    public function getOutputMessages(): array
+    {
+        return $this->outputMessageList;
+    }
+
+    /**
+     * @param array<string> $outputMessageList
+     *
+     * @return $this
+     */
+    public function setOutputMessages(array $outputMessageList)
```

Comment:
What do you think about changing it to `addMultipleOutputMessages` to prevent property override?